app/services/account_reset/grant_request.rb

```diff
@@ -7,9 +7,10 @@ def initialize(user)
     def call
       token = SecureRandom.uuid
       arr = AccountResetRequest.find_by(user_id: @user_id)
+      return if fraud_user?(arr) && fraud_wait_period_not_met?(arr)
```

Comment:
Can we update specs for this class in `spec/services/account_reset/grant_request_spec.rb` ?

In particular, I noticed this method was previously always returning a boolean and now may return `nil`. Maybe we should change this to...

```suggestion
      return false if fraud_user?(arr) && fraud_wait_period_not_met?(arr)
```